src/routes/safe/components/Transactions/TxList/SpendingLimitDetails.tsx

```diff
@@ -86,21 +92,19 @@ export const DeleteSpendingLimitDetails = ({ txData, txInfo }: SpendingLimitProp
   const txTo = getTxTo({ txInfo })
 
   return (
-    <>
-      <SpendingLimitRow>
-        <Text size="xl" strong>
-          Delete spending limit:
-        </Text>
-      </SpendingLimitRow>
-      <SpendingLimitRow>
+    <StyledTxInfoDetails>
+      <StyledDetailsTitle size="sm" strong color="placeHolder">
+        Delete spending limit:
+      </StyledDetailsTitle>
+      <StyledInfoBlock>
         <AddressInfo
           title="Beneficiary"
           address={(beneficiary as string) || txTo?.value || '0x'}
           name={txTo?.name || undefined}
           logoUri={txTo?.logoUri || undefined}
         />
-      </SpendingLimitRow>
-      <SpendingLimitRow>{tokenInfo && <TokenInfo amount="" title="Token" token={tokenInfo} />}</SpendingLimitRow>
-    </>
+      </StyledInfoBlock>
+      <StyledInfoBlock>{tokenInfo && <TokenInfo amount="" title="Token" token={tokenInfo} />}</StyledInfoBlock>
```

Comment:
The same happens here with `tokenInfo` and it rendering `<StyledInfoBlock>` all the time.